### PR TITLE
Fix/conanbuild (#65)

### PR DESCRIPTION
**Pull request by A-Alaa**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -47,7 +47,11 @@ ENDIF()
 
 IF(KHIVA_USE_CONAN)
     # Bring conan generated dependencies
-    INCLUDE(${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake)
+    if(EXISTS ${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake)
+        include(${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake)
+    else()
+        message(WARNING "The file conanbuildinfo.cmake doesn't exist, you have to run conan install first")
```
Comment (dcuestam) [inline CMakeLists.txt]: Please, add a point at the end of the line and change "doesn't" by "does not". It should be "The file conanbuildinfo.cmake does not exist, you have to run conan install first."

Comment (dcuestam) [reply CMakeLists.txt]: Please, could you find another way of showing that the conan install command should be used?  Something like "you have to run the "conan install" command first."

File: README.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Once we have installed all Khiva dependencies, we are ready to install Khiva fro
 
 - Install conan, c++ package manager, preferably running `pip install conan`. For more information and alternative installation options, please refer to [conan manual page](http://docs.conan.io/en/latest/installation.html).
 - Run `conan remote add conan-mpusz https://api.bintray.com/conan/mpusz/conan-mpusz`.
-- Create `build` folder and, after moving into the new folder, run `conan install .. --build missing`.
+- Create `build` folder and, after moving into the new folder, run `conan install .. --build google-benchmark`.
```
Comment (dcuestam) [inline README.md]: Sorry, I did not see this change. Please, maintain "missing" instead of "google-benchmark".

Comment (otorreno) [reply README.md]: Yes, otherwise Google test, boost, etc will not get installed

Comment (A-Alaa) [reply README.md]: I am sorry for that mistake!

**Comment by dcuestam** [comment]
@A-Alaa could you merge the pull request, please?

File: README.md (hunk 2 above)
Comment (otorreno) [reply README.md]: No worries! Thanks

**Pull request merged**